projects/pt1/e2e_testing/main.py

```diff
@@ -7,7 +7,7 @@
 import re
 import sys
 
-from torch_mlir_e2e_test.framework import run_tests
+from torch_mlir_e2e_test.framework import run_tests, TestOptions
```

Comment:
How do you use TestOptions?
`ImportError: cannot import name 'TestOptions' from 'torch_mlir_e2e_test.framework' `